www/package.json

```diff
@@ -1,12 +1,15 @@
 {
   "name": "www",
-  "version": "0.1.0",
   "private": true,
+  "version": "0.1.0",
+  "type": "module",
   "scripts": {
+    "dev": "vite",
```

Comment:
There is `start` already, do we need both?

Reply:
Done